.github/CODEOWNERS

```diff
@@ -0,0 +1,54 @@
+# See help here: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# These owners will be the default owners for everything in the repo.
+* @samet-akcay @harimkang
+
+# CI/CD
+/.ci/                                                       @samet-akcay @yunchu
+/.github                                                    @samet-akcay @yunchu
+
+# Docs
+/docs                                                       @samet-akcay @harimkang @yunchu
+
+# OTX config
+/src/otx/v2/config                                          @samet-akcay @harimkang @goodsong81
+
+# OTX API & CLI
+/src/otx/v2/api                                             @samet-akcay @harimkang
+/src/otx/v2/cli                                             @samet-akcay @harimkang
+
+# OTX Adapters Datumaro/Openvino
+/src/otx/v2/adapters/datumaro                               @samet-akcay @harimkang @wonjuleee
+/src/otx/v2/adapters/openvino                               @samet-akcay @harimkang @sovrasov
+
+# OTX Adapters torch modules
+/src/otx/v2/adapters/torch/modules                          @samet-akcay @harimkang @goodsong81
+
+# OTX Adapters anomalib
+/src/otx/v2/adapters/torch/anomalib                         @samet-akcay @harimkang @ashwinvaidya17
+
+# OTX Adapters mmengine
+/src/otx/v2/adapters/torch/mmengine/engine.py               @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/registry.py             @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/modules                 @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/utils                   @samet-akcay @harimkang @goodsong81
+
+# OTX Adapters mmpretrain
+/src/otx/v2/adapters/torch/mmengine/mmpretrain/engine.py    @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/mmpretrain/dataset.py   @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/mmpretrain/model.py     @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/mmpretrain/registry.py  @samet-akcay @harimkang @goodsong81
+/src/otx/v2/adapters/torch/mmengine/mmpretrain/modules      @samet-akcay @harimkang @goodsong81
```

Comment:
What about /src/otx/v2/adapters/torch/mmengine/ ?
I was asked to advise mmX section so that it would be good to add the global owners for mmengine directory.

Reply:
Thank you. Fixed. :)